src/systems/camera-system.js

```diff
@@ -213,7 +219,7 @@ export class CameraSystem {
     this.mode = NEXT_MODES[this.mode] || 0;
   }
 
-  inspect(o, distanceMod, temporarilyDisableRegularExit) {
+  inspect(o, distanceMod, temporarilyDisableRegularExit, optionalAlternativeToNotHide) {
```

Comment:
Instead of adding `optionalAlternativeToNotHide` -- add `alternatePivot` or something to indicate rotating around something else.